### PR TITLE
PR

### PR DESCRIPTION
**Pull request by sarveshrulz**

### CODE REVIEW

File: config/caf_fw.mk [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Telephony
+PRODUCT_PACKAGES += \
+    telephony-ext
```
Comment (krasCGQ) [inline config/caf_fw.mk:3]: Device-specific

File: config/common_audio.mk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# Alarms
+PRODUCT_COPY_FILES += \
+    $(ALARM_PATH)/BeyondTheDream.ogg:system/media/audio/alarms/BeyondTheDream.ogg \
+    $(ALARM_PATH)/LikeWhat.ogg:system/media/audio/alarms/LikeWhat.ogg
+
+# Notifications
+PRODUCT_COPY_FILES += \
+    $(NOTIFICATION_PATH)/Auroran.ogg:system/media/audio/notifications/Auroran.ogg \
+    $(NOTIFICATION_PATH)/CyanMail.ogg:system/media/audio/notifications/CyanMail.ogg \
+    $(NOTIFICATION_PATH)/CyanMessage.ogg:system/media/audio/notifications/CyanMessage.ogg \
```
Comment (krasCGQ) [inline config/common_audio.mk:19]: I see CM-era ringtones I go away

**Review by krasCGQ — CHANGES_REQUESTED** [review]

File: config/caf_fw.mk (hunk 1 above)
Comment (Anandnv) [reply config/caf_fw.mk:3]: Can be call it from device tree

Comment (sarveshrulz) [reply config/caf_fw.mk:3]: OK sorry, some ROMs did it though...

Comment (Anandnv) [reply config/caf_fw.mk:3]: Nhaa. We can merge it tho

**Pull request merged**